apps/docs/src/app/platform/component-docs/platform-table/platform-table-examples/platform-table-filterable-example.component.html

```diff
@@ -16,7 +16,7 @@
 <fdp-table-view-settings-dialog [table]="table">
     <fdp-table-view-settings-filter column="price" label="Price" type="custom">
         <ng-container *fdpViewSettingsFilterCustomDef="let model">
-            <fd-layout-panel>
+            <fd-layout-panel style="box-shadow: none">
```

Comment:
why do we have this?

Reply:
this is the fix for https://github.com/SAP/fundamental-ngx/issues/7008

Reply:
this is not a fix as it changes only a doc example

Reply:
@droshev but the actual issue with box-shadow comes from example. I don't think it should be handled on the level of table component, because what if developers will use any other component that has box-shadow?

Reply:
I don't think we should use `fd-layout-panel` in those examples. The main reason is that the layout panel is not a Fiori component.

Reply:
@droshev replaced with `fdp-form-group`